test/contrib/kubernetes_test.py

```diff
@@ -81,19 +81,25 @@ def test_success_job(self):
 
     def test_fail_job(self):
         fail = FailJob()
-        self.assertRaises(RuntimeError, fail.run)
+        self.assertRaises(AssertionError, fail.run)
         # Check for retrials
         kube_api = HTTPClient(KubeConfig.from_file("~/.kube/config"))  # assumes minikube
         jobs = Job.objects(kube_api).filter(selector="luigi_task_id="
                                                      + fail.job_uuid)
         self.assertEqual(len(jobs.response["items"]), 1)
         job = Job(kube_api, jobs.response["items"][0])
         self.assertTrue("failed" in job.obj["status"])
-        self.assertTrue(job.obj["status"]["failed"] > fail.max_retrials)
+        time.sleep(10)
+        self.assertTrue(
+            len(Pod.objects(kube_api).filter(selector="job-name={}".format(fail.uu_name)).response['items']) ==
```

Comment:
@Tarrasch this should still be > rather than == I think the sleep(10) is hiding this. I think the behaviour should be 1 greater than max retries since the filter will also include the original failed job. I can make that change and raise it in another PR.